src/background/index.js

```diff
@@ -1,10 +1,8 @@
 import * as raven from './raven';
-
-// @ts-expect-error - `EXTENSION_CONFIG` is missing from types
-if (window.EXTENSION_CONFIG.raven) {
-  // @ts-expect-error - `EXTENSION_CONFIG` is missing from types
-  raven.init(window.EXTENSION_CONFIG.raven);
-}
-
+import settings from './settings';
```

Comment:
This is true, although you have to be careful here since some of the imports have side effects so order may matter. In this case it was simple to rework the imports so that none of them have side effects, which allows them to be ordered as normal.